### DISCUSSION TITLE
File Extraction

### DISCUSSION THREAD

**Discussion by Sachinkadbane** [aonez/Keka#1183] [Q&A] [answered]
Hi I have bought the app from Appstore.
File extracting file I'm getting an error. Can u help me with this.
<img width="1728" alt="Screenshot 2022-11-19 at 11 51 30 AM" src="https://user-images.githubusercontent.com/118590758/202838016-9f05299f-368b-4864-aeff-72ec272dc255.png">

**Comment by aonez** [comment, marked as answer]
Hi!

You will need all the ZIP parts in the same folder to be able to extract them. So for example maybe week_02.zip has 5 parts like so:

- week_02.zip.001
- week_02.zip.002
- week_02.zip.003
- week_02.zip.004
- week_02.zip.005

You need to download all these parts and be sure to place them in the same folder. Then just opening one of them will extract the files properly.

Let me know if that worked out for you.

**Reply by Sachinkadbane** [reply]
It worked, thanks a ton for a quick reply.

**Reply by aonez** [reply]
Glad to help!
